recipes/pixell/meta.yaml

```
@@ -0,0 +1,95 @@
+{% set name = "pixell" %}
+{% set version = "0.26.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pixell-{{ version }}.tar.gz
+  sha256: a84070dfdf3701563f9fc3ea1ec454d7b71351edddb62ff51df1180f05d72ec2
+  patches:
+    - qsort_const.patch
+
+build:
+  skip: true  # [py<39 or win]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps .
+  script_env:
+    - FFLAGS=-Wno-tabs -Wno-conversion -Wno-unused-variable
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib("c") }}
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - automake
+    - autoconf
+    - m4
+    - libtool
+    - make
+  host:
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - libopenblas * *openmp*
+    - libblas * *openblas
+    - python
+    - meson-python
+    - cython
+    - numpy <2.0
+    - scipy
+    - astropy
+    - healpy
+    - matplotlib-base
+    - pyyaml
+    - h5py
+    - pillow
+    - pyfftw
+    - libxcrypt1  # [linux]
+  run:
+    - llvm-openmp  # [osx]
+    - libopenblas * *openmp*
+    - libblas * *openblas
+    - {{ pin_compatible("numpy") }}
+    - {{ pin_compatible("scipy") }}
+    - {{ pin_compatible("libopenblas") }}
+    - {{ pin_compatible("libblas") }}
```

These all have `run_exports` and thus `pin_compatible` should not be used (and actually breaks things)
```suggestion
    - {{ pin_compatible("scipy") }}
```